Makefile

```diff
@@ -7,6 +7,8 @@ goarch=amd64
 ldXflags=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=info
 ldXflagsD=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=debug
 
+MOCKERY_VERSION=v2.38.0
```

Comment:
why do we hardcode de version instead of using `latest` like on other dependencies like `go.uber.org/mock/mockgen@latest`? Is there a risk that this becomes outdated?

Reply:
Thanks for the explanation! Makes sense now